### ISSUE TITLE
Add an Answer action to University Skill 

### ISSUE THREAD

**Issue by betterclever** [fossasia/susi_skill_data#253]
**Actual Behaviour**

In the university skill, https://github.com/fossasia/susi_skill_data/blob/master/models/general/knowledge/en/university.txt
only Table Action is present.

**Expected Behaviour**

There should also be an answer action like, "Sure, here is a list of universities". Answer Action is a primary text for Speech to Text interfaces in SUSI Apps.

**Steps to reproduce it**

N/A

**LogCat for the issue**

N/A

**Screenshots of the issue**

N/A

**Would you like to work on the issue?**

If issue is relevant, I may work on this. Someone else may also take this up.

**Comment by betterclever** [comment]
@dynamitechetan @DravitLochan What are your views? We should include a answer action as well. Right?

**Comment by DravitLochan** [comment]
Yeah! add the line in front of !console: and we will be good to go.

**Comment by LoveChauhan-18** [comment]
I have added a primary answer text before the table action in the University skill to support speech-to-text interfaces.

Now the skill responds with:
"Sure, here is a list of universities in <country>."
while keeping the existing dynamic table output unchanged.

Please review the changes. Feedback is welcome.
